src/Switch/Switch.js

```diff
@@ -80,6 +100,9 @@ Switch.propTypes = {
    * The CSS class name of the root element.
    */
   className: PropTypes.string,
+  label: PropTypes.node,
+  labelClassName: PropTypes.string,
+  labelReverse: PropTypes.bool,
```

Comment:
I think that it would be great to have a `defaultProps` for the documentation generator.